.eslintrc.json

```
@@ -0,0 +1,29 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "env": {
+    "browser": true,
```

This should not have been here 🙈 a github action doesn't run in the browser. My bad...
```suggestion
    "node": true,
```